gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -0,0 +1,191 @@
+//tag::config-readiness-liveness-page-body[]
+
+Flex Gateway supports two methods for ensuring reliability and efficiency.
+
+* Readiness probes test whether a gateway instance is configured correctly and ready for incoming traffic. This ensures that end-users are not affected by unstable or partially started services. For information, refer to <<configure-a-readiness-probe>>.
```

Comment:
```suggestion
* Readiness probes test whether a gateway instance is configured correctly and ready for incoming traffic. This ensures that end-users are not affected partially configured APIs. For information, refer to <<configure-a-readiness-probe>>.
```

I'm afraid that "unstable or partially started services" might be interpreted as we would run checks to the customers services (upstreams). The readiness check only ensures that the APIs have been configured, but it says nothing of the upstream services